formatter/action.yml

```
@@ -0,0 +1,21 @@
+name: "Formatter"
```

Probably should be `Filename Formatter`.
```suggestion
name: "Filename Formatter"
```